src/tokens/universal.tokens.json

```diff
@@ -794,17 +794,17 @@
       "label": {
         "type": "custom-fontStyle",
         "value": {
-          "fontSize": 10,
+          "fontSize": 12,
           "textDecoration": "none",
           "fontFamily": "Inter Variable",
-          "fontWeight": 700,
+          "fontWeight": 400,
           "fontStyle": "normal",
           "fontStretch": "normal",
           "letterSpacing": 0,
-          "lineHeight": 10,
+          "lineHeight": 12,
           "paragraphIndent": 0,
           "paragraphSpacing": 0,
-          "textCase": "uppercase"
+          "textCase": "none"
```

Comment:
@aguscruiz, the default value for text casing on the label component is `uppercase`, but doesn't come from the tokens. If this is changing, would you mind opening an issue to request that change in the component?

Reply:
...or make the change yourself in the label component on line 77.

Reply:
Yep, those come from the tokens.